src/main/java/uk/gov/hmcts/darts/arm/component/impl/ArmResponseFilesProcessSingleElementImpl.java

```diff
@@ -99,8 +102,7 @@ private void initialisePreloadedObjects() {
     }
 
     private void processInputUploadFile(ExternalObjectDirectoryEntity externalObjectDirectory) {
-        // IU - Input Upload - This is the manifest file which gets renamed by ARM.
-        // EODID_MEDID_ATTEMPTS_6a374f19a9ce7dc9cc480ea8d4eca0fb_1_iu.rsp
+        // IU Input Upload File - EODID_MEDID_ATTEMPTS_6a374f19a9ce7dc9cc480ea8d4eca0fb_1_iu.rsp
```

Comment:
Can any context be added to this comment, as it looks a bit cryptic to the uninformed reader. What is the signifance of the `EODID_MEDID_ATTEMPTS_6a374f19a9ce7dc9cc480ea8d4eca0fb_1_iu.rsp` part?

Reply:
The filename starts with external object directory id  then the object id such as media id then the transfer attempts then some hash code, then a status then the file type so in this case its IU

Reply:
Its not the manifest file as was originally thought

Reply:
Ok, can a description to that effect be added to the actual code comment? I think that would make the intent of the comment a bit clearer.